src/.nuget/Microsoft.NETCore.Platforms/runtime.json

```diff
@@ -0,0 +1,248 @@
+{
```

Comment:
This is out of date with what we have in CoreFX for the RID graph, but if you do what @ericstj says above, you can delete this file.